icons/hardhat.svg

```diff
@@ -0,0 +1,18 @@
+<svg viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
The icon looks nice, but in my opinion it would be better if we could use the icon of the Material Design Icons package:

https://materialdesignicons.com/icon/hard-hat

Reply:
Thank you, now I like it 😊